### PR TITLE
New readme with info on running the code.

### PR DESCRIPTION
**Pull request by margocrawf**

### CODE REVIEW

**Review by HALtheWise — CHANGES_REQUESTED** [review]
There are updated versions of the rosdep lines in the Google Doc. We should include those here before merging. https://docs.google.com/document/d/1ECOGbhQr-hKkFuAEnFwf_cBLGj2PSz5y1HWLaPilLZY/edit#heading=h.bu28irn08btv

File: README.md [hunk 1]
```diff
@@ -1,5 +1,116 @@
...
+    * test by running `git --version` in a terminal
+5. [V-REP](http://www.coppeliarobotics.com/downloads.html) (educational version, ≥3.3.2)
+    * test by running `a command that is yet to be determined` in a terminal
+6. the [oars-roboboat](https://github.com/olin-robotic-sailing/oars-roboboat) git repository in your catkin_ws/src folder
+    * test by running `roscd oars_pkg` in a terminal
+7. All of the dependencies of the oars packages
+* test by running 
+```
+rosdep update
+rosdep check oars_pkg pool_server oars_arbiter
```
Comment (HALtheWise) [inline README.md]: There is a new version of this using --from-paths in the Google Doc

**Comment by HALtheWise** [comment]
Also, to prevent other things from diverging as time goes on, we should consider deleting the Google Doc content and replacing it with a link to here.

File: README.md (hunk 1 above)
Comment (margocrawf) [reply README.md]: Okay!

**Comment by margocrawf** [comment]
We should definitely try our best to prevent different versions from diverging from eachother if possible. Not sure what the solution is with the wiki and the regular repo, but for google docs I guess deleting it works?

**Review by HALtheWise — APPROVED** [review]

**Pull request merged**